qualtran/cirq_interop/jupyter_tools.py

```diff
@@ -62,7 +62,7 @@ def circuit_with_costs(circuit: 'cirq.AbstractCircuit') -> 'cirq.AbstractCircuit
     """Annotates each operation in the circuit with its T-complexity cost."""
 
     def _map_func(op: cirq.Operation, _):
-        t_cost = t_complexity_protocol.t_complexity(op)
+        t_cost = t_complexity_protocol.t_complexity(op.gate)
```

Comment:
This shouldn't be required now since getting the t-complexity out from a `GateOperation` should be supported after the recent bugfix as per my comment below. Can you verify and revert since we still have some logic to get T-complexity of operations that don't have a gate (like classically controlled ops)?